### PR TITLE
Add Ocean Fram1 Printer to print-manager

### PR DESCRIPTION
**Pull request by masterhou**
Ocean Fram1 is a delta 3d printer, FDM type, use PLA and ABS.
We want use Print Studio to slice STL objects.

### CODE REVIEW

**Comment by ghost** [comment]
Returns look fine, but we need a signed contributors agreement.
also helpful if you submit against sandbox branch

File: spark-print-data/printertypes.json [hunk 1]
```diff
@@ -662,6 +662,85 @@
...
+            "model_number": "1.0.0",
+            "icons": {
+                "16x16_id": "data/OCFarm1_16x16.png",
+                "32x32_id": "data/OCFarm1_32x32.png",
+                "50x50_id": "data/OCFarm1_50x50.png",
+                "100x100_id": "data/OCFarm1_100x100.png"
+            },
+            "technology": "FDM",
+            "default_material_id": "081011F3-4DE4-4580-9456-5F862DCAC3F0",
+            "default_profile_id": "F5641294-BD6B-44F9-8C34-1DDEB9394296",
```
Comment (ghost) [inline spark-print-data/printertypes.json]: This profile_id does not exist please use one of the ones you created in the profiles sections

**Comment by masterhou** [comment]
I have modify the profile id.

**Comment by ghost** [comment]
All Tests pass, Print Studio works with new printer. Accepting pull

**Pull request merged**